browser/farbling/brave_navigator_languages_farbling_browsertest.cc

```diff
@@ -33,7 +33,10 @@
 #include "content/public/test/content_mock_cert_verifier.h"
 #include "net/dns/mock_host_resolver.h"
 #include "third_party/blink/public/web/web_local_frame.h"
-#include "third_party/blink/renderer/core/frame/navigator_language.h"
+
+// TODO(sko) Depending on core directly isn't allowed by the visibility of the
+// target. it might be better to export it through public/ layer.
+#include "third_party/blink/renderer/core/frame/navigator_language.h"  // nogncheck
```

Comment:
do we really need this file in this test?

Reply:
I have no idea, I just guarded. Let me try again

Reply:
Yeah, you were right. Build passed without this include. Updated!